doc/6-sample/README.md

```diff
@@ -159,3 +159,14 @@ in there, you can modify certain data pertaining to your sample, such as the:
 - **Create instrument from sample**: creates a new instrument with its sample set to the current sample.
 - **Zoom**: shows and sets sample view zoom level.
 - **Zoom mode**: switches between "Auto" (entire sample fits in window) and "100%" (each horizontal pixel represents one sample point).
+
+in the sample viewer:
+- left-click and drag to select a region of the sample.
+- right-click for an editing menu:
+  - **cut**: puts the selection in the copy buffer and deletes it from the sample.
+  - **copy**: copies the selection into the copy buffer.
```

Comment:
"copy buffer" is a bit awkward. Maybe "sample clipboard"? (but that isn't good enough either)

Reply:
If we're rejecting "copy buffer", "sample clipboard" is the best I can think of for now.

Reply:
Hmmm.... The problem with "copy buffer" is that it isn't the only copy buffer in Furnace. We could use "sample copy buffer", but that's longer.

I guess "sample clipboard" is fine for now.